include/ur_api.h

```diff
@@ -963,7 +963,8 @@ urAdapterGetLastError(
 typedef enum ur_adapter_info_t {
     UR_ADAPTER_INFO_BACKEND = 0,         ///< [::ur_adapter_backend_t] Identifies the native backend supported by
                                          ///< the adapter.
-    UR_ADAPTER_INFO_REFERENCE_COUNT = 1, ///< [uint32_t] Reference count of the adapter.
+    UR_ADAPTER_INFO_VERSION = 1,         ///< [uint32_t] Specifies the adapter version, defaults to 1.
```

Comment:
maybe this should come after REFERENCE_COUNT in the yml to preserve the old enum value